compiler/dyno/include/chpl/resolution/resolution-types.h

```diff
@@ -1380,6 +1396,51 @@ class ResolvedFields {
   }
 };
 
+class ResolvedParamLoop {
+  private:
+    const uast::For* loop_;
+    std::vector<ResolutionResultByPostorderID> loopBodies_;
+
+  public:
+    using LoopBodies = std::vector<ResolutionResultByPostorderID>;
```

Comment:
IMO it would be nice to put this in its own `public:` block just before the private field decls, but it's not a big deal either way.